install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -168,6 +168,10 @@ containers:
     valueFrom:
       fieldRef:
         fieldPath: status.podIP
+  - name: SERVICE_ACCOUNT
```

Comment:
Please update this PR's description with the link to the corresponding backport PR on istio/installer.

Reply:
@douglas-reid I couldn't find any reference to the corresponding istio/installer PR and/or backport PR on istio/installer in this backport or the original https://github.com/istio/istio/pull/15551.